src/current/v25.1/time.md

```diff
@@ -151,12 +151,12 @@ Comparing `TIME` values:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> INSERT INTO time VALUES (1, TIME '05:40:00.123456'), (2, TIME '05:41:39.12345');
+> UPSERT INTO time_precise VALUES (1, TIME '05:40:00.123456'), (2, TIME '05:41:39.12345');
 ~~~
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> SELECT * FROM time;
+> SELECT * FROM time_precise;
 ~~~
 
 ~~~
```

Comment:
This is the output I get:

~~~
  time_id |   time_val
----------+----------------
        1 | 05:40:00.1235
        2 | 05:41:39.1235
~~~

Reply:
updated!